docs/index.md

```diff
@@ -19,3 +19,7 @@ release X this protocol is fully supported.
 
 [communication]: https://wiki.diasporafoundation.org/How_we_communicate
 [entities]: {{ site.baseurl }}/entities/
+
+## Implementation
+
+An implementation of this protocol is available as a Ruby on Rails Gem under the AGPL [on github](https://github.com/diaspora/diaspora_federation)
```

Comment:
"Ruby on Rails Gem" -> "a Ruby gem" (explanation: Most of the code is generic ruby in the gem `diaspora_federation` and only a very small part is rails specific in the gem `diaspora_federation-rails`. You can use the code without rails if you write the controllers yourself.)